src/components/SQFormDialog/SQFormDialog.tsx

```diff
@@ -57,7 +57,7 @@ export interface SQFormDialogProps<Values extends FormikValues> {
    * */
   validationSchema?: AnyObjectSchema;
   /** Callback function invoked when the user clicks the tertiary button */
-  onTertiaryClick?: React.MouseEventHandler<HTMLButtonElement>;
+  onTertiaryClick?: (formikContext: FormikContextType<Values>) => void;
```

Comment:
Don't forget to pass the generic from `SQFormDialog` to `SQFormDialogInner` as well

Reply:
ahh good call and thanks for explaining this a bit more!